### ISSUE TITLE
Forum: subscriptions for topic not working - notifications aren't sent

### ISSUE THREAD

**Issue by AlexTr** [unacms/UNA#1423]
Or maybe they aren't needed here at all

**Comment by AntonLV** [comment]
Subscriptions for topic should be removed because they aren't used anywhere and don't meet the idea of Subscriptions Engine in UNA.

**Comment by andreyasko** [comment]
Disagree. The idea is that users follow the interesting subject and not specific authors. Perhaps they don't participate, so normal notifications wouldn't be triggered. Should be fixed and send notifications about new replies.

**Comment by AlexTr** [comment]
@antaratma there are a lot of duplicate notifications already, it will make situation even worse. Also it isn't fitting in our system very well, since user can subscribe to profile based modules, but forum is just text post, which can't have subscribers by the design.

**Comment by andreyasko** [comment]
I agree that it's not in-line with standard "Following" that we have for profiles, but it's nonetheless significant, if not critical for most interest-based communities, where people don't care to follow or even recognise specific authors, but focus entirely on topics of conversations. Same applies to most messenger-style sites, where people join/follow conversation channels because they're interested in the topic. 

Even on Facebook, lack of such functionality results in the problem that users add "F" comment to the post they want to follow because they are interested in comments or updates, and not interested in following participants. I think it's the kind of feature which was useful and we need to fix, debug and make it work right.

**Comment by AlexTr** [comment]
@antaratma I think we can't add it now, it will result in duplicate notifications and inconsistent behaviour. I think we need to plan it for later stage when at least problem with duplicate notifications will be resolved.
